File: js/JS200/solve.js
const cipher_flag = "K@UC,bswslubr.wohp.dibokdmdb";

let i = 0
let flag = ""
for (let i = 0; i < 28; i += 4) {
	flag += String.fromCharCode(cipher_flag.charCodeAt(i) ^ 1);
	flag += String.fromCharCode(cipher_flag.charCodeAt(i + 1) ^ 3);
	flag += String.fromCharCode(cipher_flag.charCodeAt(i + 2) ^ 3);
	flag += String.fromCharCode(cipher_flag.charCodeAt(i + 3) ^ 7);
}

console.log(`flag: ${flag}`)

File: js/JS200/solve.py
#!/bin/env python3

flag = bytearray("K@UC,bswslubr.wohp.dibokdmdb", "utf-8")

for i in range(0, len(flag), 4):
	flag[i] ^= 1
	flag[i + 1] ^= 3
	flag[i + 2] ^= 3
	flag[i + 3] ^= 7

print(f"flag: {flag.decode()}")
